webextensions/api/devtools.json

```
@@ -611,6 +914,65 @@
               }
             }
           },
+          "SourcesPanel": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "createSidebarPane": {
+              "__compat": {
+                "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/SourcesPanel/createSidebarPane",
+                "support": {
+                  "chrome": {
+                    "version_added": true
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "onSelectionChanged": {
+              "__compat": {
+                "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/SourcesPanel/onSelectionChanged",
+                "support": {
+                  "chrome": {
+                    "version_added": true
```

This appears to be from before Chrome 21 as the reference to this file existed before the oldest historical entry on [chrome/common/extensions/api/devtools/panels.json](https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/extensions/api/devtools/panels.json;bpv=1).

---

Based on [this example in Chrome 13](https://github.com/chromium/chromium/commit/50cacea842493916f8793c3f4974b96f48af440d#diff-ef917f8c7954b1a690ec2174c0a6518bb7d3105fee52b422e762abb4af45db2aR23), `elements.onSelectionChanged` was there since `devtools.panel` was experimental, which is confirmed by [the commit in Chrome 18 that took it out of experimental](https://github.com/chromium/chromium/blob/2cf725f68fa5106fd5bcb1e1fb8699bb3628cafe/chrome/common/extensions/api/devtools_api.json#L198):

```suggestion
                    "version_added": "18"
```